gems/pending/appliance_console/prompts.rb

```diff
@@ -105,14 +105,6 @@ def ask_for_password_or_none(prompt, default = nil)
       ask_for_password(prompt, default).gsub(/^'?NONE'?$/i, "")
     end
 
-    def ask_for_date(prompt)
-      just_ask(prompt, nil, DATE_REGEXP)
-    end
-
-    def ask_for_time(prompt)
-      just_ask(prompt, nil, TIME_REGEXP)
-    end
-
```

Comment:
@carbonin Thank you for the input! I appreciate the help. I had viewed these methods are specific to the DateTimeConfiguration class. I had viewed prompts to contain methods used by the appliance_console.rb directly and/or shared by other classes. Can you please explain why you feel these should be in the prompts.rb?